frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -12,7 +13,13 @@ import {
 } from '@patternfly/react-icons';
 
 import { ClusterOperatorPage } from './cluster-operator';
-import { clusterChannelModal, clusterUpdateModal, errorModal } from '../modals';
+import {
+  clusterChannelModal,
+  clusterMoreUpdatesModal,
+  clusterUpdateModal,
+  errorModal,
+} from '../modals';
+import { getSortedUpdates } from '../modals/cluster-update-modal';
```

Comment:
I would move this to `cluster-settings.ts` if it's used outside the modal now.

Reply:
Done.